src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -243,6 +243,27 @@ private async Task DeleteBranchAsync(string accountName, string projectName, str
             PrStatus status,
             string keyword = null,
             string author = null)
+        {
+            return await SearchPullRequestsAsync(repoUri, status, pullRequestBranch: pullRequestBranch, keyword, author);
```

Comment:
I don't think this is right? It looks like you're passing keyword to the targetPullRequestBranch param, and author to the keyword param.

Could we just not have the overloads and you modify this function to work for your needs? it's currently dead code that is broken, so might as well make this a function that actually works?

Reply:
I would just fix this as part of the test that you're writing that needs it. There's no reason to have this PR to add more unused code.

Reply:
I added an overload because the current version is part of the required base class implementation. I can have the required implementation throw NotImplemented and make a separate method for the test case.

Reply:
We're dealing with code that is not in use, why not change it from the base class?

Reply:
Because the GitHub implementation is in use and they both come from the same base requirement

Reply:
I don't think the GitHub implementation is in use either. Or at least I couldn't find any usage. Could you point me to the one you found? if that's the case, we definitely need both to be in sync.

Reply:
Oh. I thought I found it. Let me look again.

Reply:
All the of references for the GitHub version eventually dead-end somewhere with no references, so it looks like that's safe to change. I'll take another pass and see what I can do to minimize any new additions to the code given that information.

Reply:
If the code is dead... let's just delete it? Why add tests for a thing with no callers? How do we know what the "correct" behavior is to write tests?

Reply:
Because I need to make this exact api call and my initial thinking was that it would be simpler to use the one that was already written. It's not turning out that way, so I might just do what I need in the test code.